shopfloor/actions/schema.py

```diff
@@ -48,6 +49,7 @@ def move_line(self, with_packaging=False, with_picking=False):
             "location_src": self._schema_dict_of(self.location()),
             "location_dest": self._schema_dict_of(self.location()),
             "priority": {"type": "string", "nullable": True, "required": False},
+            "progress": {"type": "float", "nullable": True, "required": True},
```

Comment:
if it is decided that it is optional,then it shouldn't be required here.

Reply:
True!

Reply:
You should add an explicit tests in test_data/test_data_detail

Reply:
There's already a test for data_picking and another one for data_picking_with_progress right? Or do you mean something else?